lib/runner.js

```diff
@@ -197,6 +199,8 @@ Runner.prototype.fail = function(test, err){
 
   if ('string' == typeof err) {
     err = new Error('the string "' + err + '" was thrown, throw an Error :)');
+  } else if (!(err instanceof Error)) {
+    err = new Error('the ' + type(err) + ' ' + stringify(err) + ' was thrown, throw an Error :)');
```

Comment:
Why is there a second branch of the conditional here?  Why not just remove the check for string, and simply display this message if `err` is not an `Error`?